docs/backup.md

```diff
@@ -0,0 +1,42 @@
+# Backup/Restore overview
+
+K0s has integrated support for backing up cluster state and configuration.
+The k0s backup utility is aiming to back up and restore k0s related parts of the cluster.
+
+The backups created by `k0s backup` command have following pieces of your cluster:
+- certificates (the content of the `<data-dir>/pki` directory)
+- etcd snapshot, if the etcd storage is used
+- k0s.yaml
+- any custom defined manifests under the `<data-dir>/manifests`
+- any image bundles located under the `<data-dir>/images`
+- any helm configuration
+
+Parts not covered by the backup utility:
+- content for any persistence volume
+- database content, in case if the `kine` is used as a storage driver
```

Comment:
kine and etcd datastores could be placed next to each other.

Reply:
They are mentioned in different sections?

Reply:
ok, I misunderstood.